locksmith/src/routes/v2/ticket.ts

```diff
@@ -39,4 +42,14 @@ router.post(
   }
 )
 
+router.get(
+  '/:network/:lockAddress/:tokenId/qr',
+  authenticatedMiddleware,
+  applicationOnlyMiddleware,
```

Comment:
Is `applicationOnly` middleware intentional? This looks like endpoint which can be used by users as well.